docs/_includes/confluence.adoc

```diff
@@ -0,0 +1,70 @@
+////
+Header: Convert Confluence XHTML to Asciidoctor
+
+Included in:
+
+- user-manual
+////
+
+You can convert Atlassian Confluence XHTML pages to Asciidoctor using this http://www.groovy-lang.org/download.html[Groovy] script.
+
+The script calls the http://pandoc.org/[Pandoc] tool to convert single or multiple HTML files exported from Confluence to AsciiDoc files (using the Python version of the AsciiDoc toolchain).
```

Comment:
Does pandoc ship the "backends" with the tool? I wanted to make the
distinction here that it isn't Asciidoctor, hence the statement.

Just as ffmpeg uses various codec libraries, does pandoc access different
transform libraries?

No worries about the location: it's an include so we can put it where we
want. 😉

Because this issue was about documenting the script, I think that it will
be complete once I remove the statement about python AsciiDoc.

Restructuring the content into a tooling guide is a good move but requires
another issue IMHO.

On Tue, Feb 16, 2016, 18:05 Dan Allen <notifications@github.com> wrote:

> In docs/_includes/confluence.adoc
> <https://github.com/asciidoctor/asciidoctor.org/pull/491#discussion_r52978101>
> :
>
> > @@ -0,0 +1,70 @@
> > +////
> > +Header: Convert Confluence XHTML to Asciidoctor
> > +
> > +Included in:
> > +
> > +- user-manual
> > +////
> > +
> > +You can convert Atlassian Confluence XHTML pages to Asciidoctor using this http://www.groovy-lang.org/download.html[Groovy] script.
> > +
> > +The script calls the http://pandoc.org/[Pandoc] tool to convert single or multiple HTML files exported from Confluence to AsciiDoc files (using the Python version of the AsciiDoc toolchain).
>
> I don't think it's true that this script uses AsciiDoc Python. I think it
> just uses pandoc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/pull/491/files#r52978101>.
>
-- 

Sent from Mobile.